Documentation/general/tool-nuget-package-format.md

```diff
@@ -0,0 +1,26 @@
+Tool NuGet package format
+-------------------------------------------
+
+Use `dotnet pack` with `PackAsTool` property is the most straight forward way. However, if your build process is highly customized and `dotnet publish` cannot create a proper distribution for you. You can create tool NuGet package using nuspec and explicitly place assets in NuGet package under the following rules.
+
+- Tool NuGet package does not have any other folder under root like _/lib_, _/content_, it only has _/tools_ folder.

```

Comment:
- The NuGet package has only _/tools_ folder under the root and does **not** contain any other folders; do not include folders like  _/lib_, _/content_, etc.

Reply:
I used this version